pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotAdministrator.java

```diff
@@ -102,10 +102,13 @@ public void execute(String[] args)
     }
   }
 
-  public static void main(String[] args)
-      throws Exception {
+  public static void main(String[] args) {
     PinotAdministrator pinotAdministrator = new PinotAdministrator();
-    pinotAdministrator.execute(args);
+    try {
+      pinotAdministrator.execute(args);
+    } catch (Exception e) {
+      System.exit(1);
```

Comment:
May be log the message/exception?

Reply:
Actually the exception will be caught in execute() method. Will let this method do logging, and let main() function check status and do exit.